templates/helm/orchestration-workflows/encode/templates/encode.yaml

```diff
@@ -70,6 +70,15 @@ spec:
       subnetName: monster-processing-network
       workerAccount: dataflow-runner@{{ $project }}.iam.gserviceaccount.com
       useFlexRS: {{ eq .Values.env "prod" }}
+    bigquery:
+      stagingData:
+        project: {{ $project }}
+        datasetPrefix: monster_staging_data
+        description: Dataset used by the Monster team for ingest ETL
+        # 7 days in units of seconds
+        expiration: '604800'
+      jadeData:
+        project: {{ if (eq .Values.env "dev") }}broad-jade-dev-data{{ else }}broad-datarepo-terra-prod-encode{{end}}
```

Comment:
Is this a good project name for prod? I just based it off of the name we used for clinvar.

Reply:
I think that's fine. We can always change it later if it auto-generates to something else.

Reply:
Oh! It would be helpful if I actually initialized the dataset in dev, so there's something to join against. I'll do that now.

Reply:
broad-jade-dev-data is correct, the dataset is in place now